dataEng_container_tools.egg-info/PKG-INFO

```diff
@@ -1,239 +1,242 @@
 Metadata-Version: 2.1
 Name: dataEng-container-tools
-Version: 0.6.1
+Version: 0.6.4
 Summary: A package containing tools for data engineering containers.
 Home-page: https://github.com/colpal/dataEng-container-tools
 Author: Alexander Saff
 Author-email: alexander_saff@colpal.com
```

Comment:
can we add a `Maintainer-email` field and include DE team google group?

Reply:
> hmm is there no way to sync `PKG-INFO` with the info under `setup.py`?

they should be synced. where do you see discrepancies?

Reply:
> I think PKG-INFO, being an egg file, is generated off of setup.py. I would think we can remove it.

@mourtallah What information in the PKG-INFO do you think shouldn't be in the repo?

Reply:
> can we add a `Maintainer-email` field and include DE team google group?

@GoodToast I've added the maintainer email